drivers/bluetooth/hci/userchan.c

```diff
@@ -64,7 +75,7 @@ static struct net_buf *get_rx(const uint8_t *buf)
 	switch (buf[0]) {
 	case H4_EVT:
 		if (buf[1] == BT_HCI_EVT_LE_META_EVENT &&
-		    (buf[3] == BT_HCI_EVT_LE_ADVERTISING_REPORT)) {
+			(buf[3] == BT_HCI_EVT_LE_ADVERTISING_REPORT)) {
```

Comment:
Nit: We normally avoid doing style changes in lines we are not modifying otherwise.
(note tabs are 8 characters long in Zephyr, so this is worse off with the change)